apps/allocations/app/components/App/AllocationsHistory.js

```diff
@@ -123,6 +123,10 @@ const Amount = styled.div`
   font-weight: 600;
 `
 
+const RecipientContainer = styled.div`
+  width: 50%;
```

Comment:
How does this look on mobile?

Reply:
@chadoh this looks okay for me. The location of the detail expansion in this block bothers me a bit, but that's not enought to prevent a merge in my opinion

![image](https://user-images.githubusercontent.com/17910833/67599918-afb9e100-f736-11e9-9e74-745df723e888.png)